sandbox/modules/identity/src/main/java/org/opensearch/identity/authz/IndexNameExpressionResolverHolder.java

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
This class is not being used in this PR, but there is some commented out code that uses this. This is a holder for the `IndexNameExpressionResolver` that is passed to `IdentityPlugin.createComponents`. This class will come in handy if resolving index patterns -> concrete index names is needed.

Reply:
So good to just ignore this for now?

Reply:
Yes, it relates to a commented out section in `OpenSearchPermission` that does index name -> concrete indices resolution and I anticipate will be used quite a bit as privilege evaluation is expanded upon.